local/checkoutservice.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
```

Comment:
do you mean to have the full checkoutservice YAML in `local` here?

Reply:
Nope. Good catch. I've cleaned it up